### PR TITLE
health checks should consider pop request times relative to the ad render times

### PR DESCRIPTION
**Pull request by hkaya**

### CODE REVIEW

**Comment by natlownes** [comment]
+1

File: test/index_spec.coffee [hunk 1]
```diff
@@ -271,10 +271,12 @@ describe 'AdView', ->
         isConnected: (cb) -> cb true
       @view.proofOfPlay.lastRequestTime = 1000
       @view._isRunning = true
+      @view._lastAdRenderTime = 1000
       @view.onHealthCheck (res) =>
         expect(res.status).to.be.true
         @clock.tick (@config.healthCheck.lastPopRequestTimeThreshold + 2000)
         @view._lastRunTime = new Date().getTime()
```
Comment (markchadwick) [inline test/index_spec.coffee]: http://i.imgur.com/eUATH77.jpg

new Date().getTime()
new Date().getTime()
new Date().getTime()
new Date().getTime()

Comment (markchadwick) [inline test/index_spec.coffee]: But seriously, are all these variables tracking the same thing? Does this need a step back?

Comment (hkaya) [inline test/index_spec.coffee]: it really looks weird but they are not tracking the same thing :) I'm setting them to some value to simulate the view workflow. on a real run, they will not be equal to the same thing but they are expected to be close (at least lower than the thresholds). for tests, it doesn't matter if we set it to the same values.

Comment (hkaya) [inline test/index_spec.coffee]: @markchadwick If you are concerned about the tests I can rewrite them in a separate commit. For now, I'd like to merge this change and hand over apps to Titan as soon as possible. What do you think?

**Comment by markchadwick** [comment]
Yeah, I'm good on this. :+1:

**Comment by hkaya** [comment]
Thank you for the review guys!

**Pull request merged**